No csd linux by default

sbatten opened the pull request.

bpasero commented on src/vs/platform/windows/common/windows.ts:
```diff
@@ -265,6 +265,11 @@ export function getTitleBarStyle(configurationService: IConfigurationService, en
 		return 'native'; // not enabled when developing due to https://github.com/electron/electron/issues/3647
 	}
 
+	const accessibilityMode = configurationService.getValue('editor.accessibilitySupport');
```
@sbatten get this setting only when `!isMacintosh` ?

bpasero commented on src/vs/platform/windows/common/windows.ts:
```diff
@@ -265,6 +265,11 @@ export function getTitleBarStyle(configurationService: IConfigurationService, en
 	}
 
+	const accessibilityMode = configurationService.getValue('editor.accessibilitySupport');
+	if (!isMacintosh && accessibilityMode) {
```
@sbatten the value of `editor.accessibilitySupport` is a string, so how does your check here actually work?

bpasero commented on src/vs/workbench/browser/parts/titlebar/menubarControl.ts:
```diff
@@ -203,39 +199,41 @@ export class MenubarControl extends Disposable {
-					run: () => {
-						this.storageService.store('menubar/electronFixRecommended', true, StorageScope.GLOBAL);
-					}
+		const message = nls.localize('menubar.linuxTitlebarRevertNotification', "We have updated the default title bar on Linux to use the native setting. If you prefer, you can go back to the custom setting ");
```
@sbatten  why is there a trailing whitespace at the end of the sentence (`you can go back to the custom setting `)?

bpasero commented:
@sbatten also, I would inline the "More Info" to be a link within the notification and not have it as extra button to reduce clutter. You can use markdown style styntax for that.

sbatten commented:
@bpasero, updated to use inline link as suggested and switched back to Open Settings for consistency. I kept the behavior to not keep  showing this message even without the user's input so as to not but the user. The notifications are not critical unlike the previous notification to address an electron bug.

bpasero commented:
@sbatten LGTM
